index.js

```diff
@@ -2,6 +2,7 @@ import config from './config/config';
 import app from './config/express';
 /* eslint-disable no-unused-vars */
 import db from './config/sequelize';
+import logger from './config/winston';
 
 const debug = require('debug')('amida-api-boilerplate:index');
```

Comment:
## Immediate recommendation (blocking)
- Disable debug logging in production
- Right now, debug logging (e.g. postgres), happens for both production and development environments to stdout. 
- Why: All logging in stdout (at least in prod) should be JSON structured

Confirm with
```
$ work@Hargrove:~/git/amida-tech/amida-messaging-microservice (logging-updates)$ NODE_ENV='production' yarn start
...
Executing (default): CREATE TABLE IF NOT EXISTS "Users" ("id"  SERIAL , "username" VARCHAR(255) NOT NULL, "createdAt" TIMESTAMP WITH TIME ZONE NOT NULL, "updatedAt" TIMESTAMP WITH TIME ZONE NOT NULL, PRIMARY KEY ("id"));
Executing (default): SELECT i.relname AS name, ix.indisprimary AS primary, ix.indisunique AS unique, ix.indkey AS indkey, array_agg(a.attnum) as column_indexes, array_agg(a.attname) AS column_names, pg_get_indexdef(ix.indexrelid) AS definition FROM pg_class t, pg_class i, pg_index ix, pg_attribute a WHERE t.oid = ix.indrelid AND i.oid = ix.indexrelid AND a.attrelid = t.oid AND t.relkind = 'r' and t.relname = 'Users' GROUP BY i.relname, ix.indexrelid, ix.indisprimary, ix.indisunique, ix.indkey ORDER BY i.relname;
```

## Next step
May want to discuss what to do with non-JSON structured debug logging in development.
While turning-off in prod is within the scope of this PR, there could be some debate about what to do in dev.
I'm inclined towards **always JSON**, with folks separating the prettification-concern, but the community seems split. From what I can tell, Winston folks bake prettificiation into the application for debug, while bunyan folks prettify seperately (all JSON in both prod and dev). YMMV.